lib/jekyll/commands/draft.rb

```diff
@@ -3,51 +3,49 @@ module Commands
     class Draft < Command
       def self.init_with_program(prog)
         prog.command(:draft) do |c|
-          c.syntax 'draft NAME'
-          c.description 'Creates a new draft post with the given NAME'
+          c.syntax syntax
```

Comment:
I was planning on having the unified init_with_program, but it seemed to cause problems with the plugin loading. I'll inline them.